api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -121,3 +121,20 @@ def test_create_portal_metadata(self, mock_upload_output_file):
                     [row[common.LIBRARY_ID_KEY] for row in rows if common.LIBRARY_ID_KEY in row]
                 )
                 self.assertEquals(output_libraries, expected_libraries)
+
+    @patch("scpca_portal.management.commands.create_portal_metadata.s3.delete_output_file")
+    def test_only_one_computed_file_at_any_point(self, mock_delete_output_file):
+        # Set up the database for test
+        self.load_test_data()
+        # First call to create the portal metadata computed file
+        computed_file_one = self.processor.create_portal_metadata(
+            clean_up_output_data=False, update_s3=True
+        )
+        # Second call to create the portal metadata computed file
+        self.processor.create_portal_metadata(clean_up_output_data=False, update_s3=True)
+        mock_delete_output_file.assert_called_with(
+            computed_file_one.s3_key, computed_file_one.s3_bucket
+        )
+        # Make sure pre-existing computed_file has been deleted and only one exists
+        existing_computed_file = ComputedFile.objects.filter(portal_metadata_only=True)
+        self.assertEqual(existing_computed_file.count(), 1)
```

Comment:
1. lets call the command twice without storing the return value
2. move the existing_computed_files count check above the mock_delete_ouput_file check
3. grab what should be the only computed file for portal metadata and assert that mock_delete_ouput_file was called with that object's values